src/main/MQ2ImGuiConsole.cpp

```diff
@@ -1055,7 +1087,11 @@ class ImGuiConsole
 		contentSize.y -= footer_height_to_reserve;
 
 		m_zepEditor->Render("##ZepConsole", contentSize);
-
+		// Is Mouse over Zep Editor
+		imgui::ImGuiZepEditor obj;
+//		s_isMouseInFadeDelay = TimedMouseHoveredEvent(obj.IsHovered());
+		s_isMouseInFadeDelay = TimedMouseHoveredEvent(ImGui::IsWindowHovered());
+		ImGui::EndChild();
```

Comment:
I Keep trying but this Works This way but does not work With the comment line above it, I know this is a scope issue, but i have not been able to access it either Directly or through ImGuiConsole::IsHovered() .. This seems to always return true